make/hotspot/lib/CompileJvm.gmk

```diff
@@ -158,6 +158,10 @@ ifeq ($(call isTargetOs, windows), true)
     WIN_EXPORT_FILE := $(JVM_OUTPUTDIR)/win-exports.def
   endif
 
+  ifeq ($(SHIP_DEBUG_SYMBOLS), public)
+    JVM_CFLAGS += -DHAS_STRIPPED_DEBUGINFO
+  endif
```

Comment:
But IIUC building the stripped pdb files doesn't necessarily mean you have deployed them in your image.

Also isn't there a way to define file specific flags so this only gets set for whitebox.cpp?